nginx.conf.d/berkeley-production.conf

```diff
@@ -1,8 +0,0 @@
-# Configuration for Berkeley hosts
```

Comment:
These files were condensed from 4 files to 2.

One file for production and one for staging. It seems easier to maintain and understand. They still share relevant configuration via includes.